common/src/supabase/groups.ts

```diff
@@ -1,5 +1,15 @@
 import { SupabaseClient } from '@supabase/supabase-js'
 import { run } from './utils'
+import { Group } from 'common/group'
+
+export async function getGroup(groupId: string, db: SupabaseClient) {
```

Comment:
Prefer putting `db` as the first argument (I think that's the convention elsewhere)